### DISCUSSION TITLE
additional profile and background details

### DISCUSSION THREAD

**Discussion by flanaganj42** [community/community#123214] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I stumbled onto your github repository and noticed your name and working in Kansas City.  It made me curious if you had any family in/from the Des Moines, IA area?  I attended [redacted by moderator].  Sorry, if this was a bother but I had to ask about fond ole memories.

**Comment by ebndev** [comment]
Hey @flanaganj42, thank you for being a part of the GitHub Community!

Please note: GitHub Discussions is a public forum and not intended to be used as a private message board. As a result, I will be locking this post but please feel encouraged to post again in the future if you have questions or topics you'd like to share with others.

I also removed personal identifying information from your post.

Thanks for understanding!
